app/javascript/api/types/rails/listings/RailsUnit.d.ts

```diff
@@ -21,10 +21,10 @@ type RailsUnit = {
   Unit_Square_Footage?: number
   attributes: ListingAttributes
   isReservedCommunity: boolean
-  Price_Without_Parking?: number
-  Price_With_Parking?: number
-  HOA_Dues_Without_Parking?: number
-  HOA_Dues_With_Parking?: number
+  Price_Without_Parking?: any
```

Comment:
What are the possible types for these fields? I am curious if we can be more specific than the any type. Could it be number | string | undefined | null ?